src/components/textarea/Textarea.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, ReactElement, ChangeEvent } from 'react';
+import cn from 'classnames';
+
+import './textarea.scss';
+
+interface ITextarea {
+  className?: string,
+  name?: string;
+  placeholder?: string;
+  value?: string;
+  rows?: number;
+  onChange?(e: ChangeEvent<HTMLTextAreaElement>): void;
+}
+
+const Textarea: FC<ITextarea> = ({
+  className,
+  name,
+  placeholder,
+  rows,
+  value,
+  onChange,
+}): ReactElement => (
```

Comment:
🧐
Лучше, когда объявление функции не занимает больше одной строчки
```suggestion
const Textarea: FC<ITextarea> = (props): ReactElement => {
  const {
    className,
    name,
    placeholder,
    rows,
    value,
    onChange,
  } = props;
```